pkg/edgeview/src/edge-view.go

```diff
@@ -540,7 +543,7 @@ func initpubInfo(logger *logrus.Logger) pubsub.Publication {
 }
 
 func sendKeepalive() {
-	ticker := time.NewTicker(120 * time.Second)
+	ticker := time.NewTicker(90 * time.Second)
```

Comment:
I thought some NAT boxes have a 30 second timer. Should this be a configurable edgeview.keepalive.timeout setting?

Reply:
I think NAT timeout is fine here, as long as the process send out new packet, even it uses a new port assignment, it's not a problem. But here I change from 120 -> 90 is because I think for 120, the zedcloud side proxy server will timeout. for 90 it's ok. Want to send less keepalive packets as possible. Yes, if we see a need, we can later add a configitem for this.

Reply:
This issue here was something annoying, while i'm in the tcp session, i spend time to search for something else, and come back to it, it timed out. i had to relaunch the session.

Reply:
Ok